src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -114,4 +127,28 @@ public boolean doMonitor(ServiceEmitter emitter)
 
     return true;
   }
+
+  private void emitGcMetrics(ServiceEmitter emitter)
+  {
+    ImmutableMap<String, Long> newGcCounters = ImmutableMap.of(
+      "jvm/gc/young/cpu", youngGcCpu.getNanos(),
```

Comment:
1) Right, this makes sense. Implemented. Thank you.

2) Disagree. "cpu" is a commonly used term for "cpu time", including in existing metrics (sys/cpu, jvm/cpu). "time" is vague.